### ISSUE TITLE
Hildon-Control-Panel (hcp) display size independence (cosmetic issue)

### ISSUE THREAD

**Issue by android-808** [maemo-leste/bugtracker#123]
Small cosmetic issue when viewing opening the settings app on displays with widths different to the N900.

Settings are divided into sections with a seperator between each like:

----- Networking -----

On N900, seperator fills the width of the screen.  In default 20180308 VM image, title and seperator are left-aligned and look a little odd.

Obviously, this should be very very low priority at the moment but a consistant look across devices is desirable.

**Comment by MerlijnWajer** [comment]
Yeah, I also noticed this. I suppose it shouldn't be too hard to fix with someone with some gtk knowledge.
